### ISSUE TITLE
TXT/CTS-CN-F2F-2019-11-15-1422.txt

### ISSUE THREAD

**Issue by MG623** [MagicHub-io/Magic-Data-ASR-SD-Challenge#8]
Hello, label about CTS-CN-F2F-2019-11-15-1422 (TXT/CTS-CN-F2F-2019-11-15-1422.txt) :

[746.317,751.504]	G00000248	男,普通话	呃，让电动车呢给人们带来利益，而不是带来了无尽的坏处
[751.873,756.208]	G00000248	男,普通话	嗯，确实还有一个问题就是现在冬天快要来临
These two sentences are actually said by two people, but labels are the same person

[1501.420,1505.677]	G00000249	男,普通话	但是说还有很多的电动车呢，它没有办法说是
[1507.849,1512.009]	G00000248	男,普通话	所以呢，大家一定要防范偷车贼的这样一个产生
These two sentences are actually said by the same person, but they are marked by different people

To sum up, the label of 751.873-1505.677 is reversed, resulting in 40% DER forever
